docs/core/tools/dotnet-new.md

```diff
@@ -318,6 +317,10 @@ List all templates available for MVC:
 
 `dotnet new mvc -l`
 
+Install version 2.0 of the Single Page Application templates for ASP.NET Core (command option available for .NET Core SDK 2.0 and later versions only):
```

Comment:
The -i option has been available with this syntax since the 1.0 CLI. I may just be reading this wrong though.

Reply:
You're reading that right. Ok that makes other portions of this doc wrong then 😢 

It seems that it was introduced in 1.1 actually.

1.0.4 options:
![image](https://user-images.githubusercontent.com/12971179/37742894-c8c40456-2d24-11e8-80c8-e7de62c1de59.png)

1.1 options:
![image](https://user-images.githubusercontent.com/12971179/37742919-eaeb622c-2d24-11e8-833c-d10fc5a4c5c2.png)